its/src/test/java/com/sonar/it/scanner/msbuild/utils/TestUtils.java

```diff
@@ -79,7 +79,7 @@ public static String getScannerVersion(Orchestrator orchestrator) {
 
   private static MavenLocation getScannerMavenLocation(String scannerVersion, ScannerClassifier classifier) {
     String groupId = "org.sonarsource.scanner.msbuild";
-    String artifactId = "sonar-scanner-msbuild";
+    String artifactId = "sonar-scanner";
```

Comment:
This is the artifact ID set in the main pom file. It is used to look for the package in the correct location.